.github/workflows/go.yml

```diff
@@ -8,11 +8,11 @@ on:
 
 jobs:
 
-  build:
-    name: Build
+  test:
+    name: Unit Test
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@master
```

Comment:
Is there not a recent release that we can use?

Reply:
fixed